docs/src/ComboBox.doc.js

```diff
@@ -0,0 +1,732 @@
+        description: 'Callback when you focus on the component',
+      },
+      {
+        name: 'onKeyDown',
+        type: '({ event: SyntheticKeyboardEvent<HTMLTextAreaElement>, value: string }) => void',
+        description: 'Callback for key stroke events. See [tags](#TYags) variant to learn more.',
+      },
+      {
+        name: 'onSelect',
+        type: '({ event: SyntheticInputEvent<>, value: string }) => void',
+        description: 'Callback when an item is selected',
+      },
+      {
+        name: 'placeholder',
+        type: 'string',
+        description: 'Specify a short description suggestion the expected input for the field',
+      },
+      {
+        name: 'selectedOption',
+        type: '{| label: string, value: string, subtext: string |}',
+        description:
+          'The selected option in ComboBox for controlled components. See [controlled ComboBox](#Controlled-vs-Uncontrolled) variant to learn more.',
+      },
+      {
+        name: 'size',
+        type: '"md" | "lg"',
+        description:
+          'Defines the height of ComboBox: md: 40px, lg: 48px. Width is defined by parent component.',
+        defaultValue: 'md',
+      },
+      {
+        name: 'tags',
+        type: 'Array<Element<typeof Tag>>',
+        description:
+          'List of tags to display in the component. See [tags](#tags) variant to learn more.',
+      },
+      {
+        name: 'ref',
+        type: "React.Ref<'input'>",
+        description:
+          'Forward the ref to the underlying component container element. See [focus management](#Ref) variant to learn more',
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection name="Best practices">
+    <MainSection.Subsection columns={2}>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use ComboBox to allow the user to edit or copy the textfield input values to select and/or narrow down from a given list of options."
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use ComboBox for a simple list of items. Use [SelectList](/SelectList) instead for the added native mobile functionality."
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`ComboBox requires \`accessibilityShowButtonLabel\` and \`accessibilityClearButtonLabel\``}
+    />
+    <MainSection.Subsection
+      title="Keyboard interaction"
+      description={`
+    * Hitting \`Enter\` or \`Space\` key on the ComboBox's trigger opens the options list
+    * \`Escape\` key closes the options list, while moving focus back on the ComboBox's trigger
+    * Arrow keys are used to navigate items within the options list
+    * \`Enter\` key selects an item within the options list
+    * \`Tab\` or \` Shift + Tab\` close the options list and move focus accordingly
```

Comment:
Could probbaly add one that "Once an item is selected, hitting Enter or Space on the clear button clears the selection and returns focus to the input

Reply:
DONE